src/multiParamManager.js

```diff
@@ -96,11 +114,17 @@ class MultiParamManager {
     }
   }
 
-  createMultiFields () {
+  createMultiFields (uniqueId) {
     if (this.funABI.inputs) {
+      var that = this
       return yo`<div>
-        ${this.funABI.inputs.map(function (inp) {
-          return yo`<div class="${css.multiArg}"><label for="${inp.name}"> ${inp.name}: </label><input placeholder="${inp.type}" title="${inp.name}"></div>`
+        ${this.funABI.inputs.map(function (inp, idx) {
+          function setUpValidator (id) {
+            $(`#${id}`).off('blur')
+            $(`#${id}`).blur(() => doValidation(that, id))
+          }
+          var id = `${uniqueId}${inp.type}${inp.name}${idx}`
```

Comment:
Are you using `uniqueId` because of the `$(`#${id}`).addClass(css.oops.className)` in `doValidation`
If so I think doing: 
`onclick=${function() { setUpValidator(this) }}`
would be better as you don't need to rely on the `id`